vignettes/manual/BuildHector.Rmd

```diff
@@ -135,37 +135,34 @@ Run Hector from the terminal with the command line.
 ./src/hector ./inst/input/name-of-ini.ini
 ```
 
-
-
-## Xcode (Mac OSX)
+## Xcode (Mac OS X)
 
 These directions assume a basic familiarity with Xcode and Mac OS X software installation.
 
 * Install [Xcode](https://developer.apple.com/xcode/downloads/) if necessary. Hector has been built and tested with Mac OS >= 10.8.5. The project files are for Xcode 11.3.1.
 * Download and install Boost, following instructions above.
 * Download the [Hector zip file](https://github.com/JGCRI/hector/archive/master.zip) or check out the repository using Git.
-* From Xcode, open the project file in `project_files/Xcode/hector.cxodeproj`.
-* Build the project. See below if you encounter errors.
-* Change the current Scheme settings (Scheme->Edit Scheme) and add a command-line argument (*Arguments* tab, e.g. "input/hector_rcp45.ini").
-* Run!
+* From Xcode, open the project file in `project_files/Xcode/hector.xcodeproj`.
 
-Xcode Build Settings to check if you encounter build errors, they should match the version installed on your machine:
+The Xcode project file is configured to use constants defining custom paths, rather than hard-coded paths.
+This means that developers and users can set these paths without overwriting each other.
 
-* *Architectures-Base SDK*: OS X 10.8 [or OS X version on machine] 
-* *Build Options-Compiler for C/C++/Objective-C*: Default compiler (Apple LLVM 5.1)
-* *Search Paths-Header Search Paths*: "/usr/local/include /usr/local/lib/boost_1_52_0/" [or other header directories, this is important for homebrew users]
-* *Search Paths-Library Search Paths*: "/usr/local/lib/" [or other library directories]
-* *LLVM-Language-C++-C++ Standard Library*: libc++ (LLVM C++ standard library with C++11 support)
-* *Linking-Other Linker Flags*: "-lgsl -lgslcblas -lm"
-* *User-Defined*: "GCC_MODEL_TUNING" defined as "G5"
+* You will need to set several custom paths (Preferences -> Locations -> Custom Paths): `BOOSTLIB`, `BOOSTROOT`, and `HECTORDIR`. These are the Boost runtime library, the Boost root under which the headers can be found in `boost/`, and the runtime working directory for the model (the repository root).
```

Comment:
I've followed these instructions but am running into issues with the boost finding some of the header files. 


<img width="1366" alt="Screen Shot 2021-06-24 at 11 16 18 AM" src="https://user-images.githubusercontent.com/27299759/123288689-cac98f00-d4dd-11eb-99e4-0322f77b8606.png">


BOOSTROOT = `/usr/local/Cellar/boost/1.72.0_3/include` 

and that `BOOSTROOT/boost/math/tools/polynomial.hpp` does exist on my machine.... what am I doing wrong?? :(

Reply:
Verify that in $BOOSTROOT there is a `boost/` subdirectory—there is? And, inside of that, all the Boost hpp files? If not, you're pointing it to the wrong place.

If yes, you may need to revisit the build setting to force it to 'see' the proper location? Hmm.

Reply:
Okay so it turns out that you have to quit xcode and restart it in order for the custom paths to take effect 🤦‍♀️